django_rq_dashboard/views.py

```diff
@@ -78,8 +78,14 @@ def serialize_scheduled_queues(queue):
 
 class SuperUserMixin(object):
     def dispatch(self, request, *args, **kwargs):
-        if not request.user.is_superuser:
-            raise PermissionDenied
+        if not request.user.is_authenticated():
+            return redirect('{}?{}'.format(reverse('admin:login'),
+                                           urllib.urlencode({
```

Comment:
This fails on Python 3. Could you update the urllib import to use `django.utils.six.moves.urllib`?